src/test/java/uk/gov/digital/ho/proving/income/audit/statistics/PassStatisticsCalculatorTest.java

```diff
@@ -27,84 +27,84 @@ public void result_givenFromDate_inResult() {
         List<AuditResultByNino> someList = emptyList();
 
         LocalDate fromDate = LocalDate.of(2019, 2, 3);
-        assertThat(accumulator.result(someList, fromDate, SOME_DATE).fromDate()).isEqualTo(fromDate);
+        assertThat(accumulator.result(someList, emptyList(), fromDate, SOME_DATE).fromDate()).isEqualTo(fromDate);
     }
 
     @Test
     public void result_givenToDate_inResult() {
         List<AuditResultByNino> someList = emptyList();
 
         LocalDate toDate = LocalDate.of(2019, 3, 4);
-        assertThat(accumulator.result(someList, SOME_DATE, toDate).toDate()).isEqualTo(toDate);
+        assertThat(accumulator.result(someList, emptyList(), SOME_DATE, toDate).toDate()).isEqualTo(toDate);
     }
 
     @Test
     public void result_emptyList_emptyResult() {
-        assertThat(accumulator.result(emptyList(), FROM_DATE, TO_DATE))
+        assertThat(accumulator.result(emptyList(), emptyList(), FROM_DATE, TO_DATE))
```

Comment:
I don't understand the significant of someList and emptyList when they are both the same?

Reply:
This is unclear, I agree. I have fixed it.